### ISSUE TITLE
Draft for Reps Resources Training

### ISSUE THREAD

**Issue by couci** [mozilla/Reps#113]
## Goal: 
Create a draft doc with training for  Reps Resources 
## Related Issues & Links: 
https://github.com/mozilla/Reps/issues/100

## Roles:

Responsible: @couci

Accountable: @couci

Supporting: Review Team

Consulted: Council, Reps

Informed: 

## Required: 
- [x] Create the draft (owner @couci) 
- [ ] Gather feedback (owner @couci, @kelimuttu ) 
- [ ] Publish online (owner@couci)

**Comment by couci** [comment]
Here is the [draft](https://docs.google.com/document/d/1VU_DXBybvcaHzl43BxlXgIJ1UxEPNgc14IT9409v2nM/edit#) and is based on previous work done by @francjp and can be found [here](https://docs.google.com/document/d/1dLqkzm8hWd-592qKIoE-QM8O8jewrytuSNXJtEV5S6g/edit)
On the draft we are missing the final sector, feedback are welcome

**Comment by couci** [comment]
Draft is finished :)

**Comment by couci** [comment]
@deimidis I would love your feedback in here since you've been leading the coaching training at the past. Also @Mte90 @dvigneshwer your feedback would be really valuable considering your involvement in the past

**Comment by Mte90** [comment]
the doc is in read-only mode :-/

**Comment by couci** [comment]
apologies for that, fixed now :)

**Comment by MichaelKohler** [comment]
The initial version has been put on Teachable. We need to make it a little bit more beautiful as it's quite a lot of text. We will do that with pictures etc. Once that is done, we can move ahead and open the track for applications. This will need a communication piece, but we should track that in another issue once the training is finished.

**Comment by barrosgg** [comment]
Hi @couci, if in the future you are interested in translating this training into other languages as Portuguese from Brazil, I can help in this, maybe it is important to translate in the future because not only are representatives dealing with resources, but community volunteers are helping representatives to work with the resources too, then some volunteers could learn one or more relevant parts of this training.

This would not be now, but in the future we can try and experiment.

**Comment by MichaelKohler** [comment]
Draft is done, we will test it with a few selected Reps first before going broad. Future tasks tracked in #168
